ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -72,7 +72,7 @@ def test001_cancel_button_edit_no(self, save):
 
     # will continue with us
     @parameterized.expand(['save_btn', 'cancel'])
-    def test002_cancel_button_edit_contact(self, save):
+    def test002_update_contact_with_save_cancel_btn(self, save):
```

Comment:
Run this test case in travis

Reply:
@0xIslamTaha this test case old one and this PR not test this test case I merged all from the master branch so this test case now same as the master branch , I mean I prefer to create a new PR for it with new branch because It need to re-implement it from A to Z and using also the API

Reply:
@dianaeldemerdash I can see that this PR updated it. No?
![image](https://user-images.githubusercontent.com/17981089/79972366-204b7b80-8496-11ea-917c-ee5fafd2aa84.png)

Reply:
@0xIslamTaha it was because the base branch from old branch not from the master branch I updated the base to be from the master branch but the PR it self no related for this test case